changelog.d/729.misc

```diff
@@ -0,0 +1 @@
+Correctly attribute mkarg for documentation contributions. We also reverted commits and recommited them with the correct Author value set.
```

Comment:
As long as we're editing `CHANGELOG.md`, this entry can be put in there manually as well.

Plus, if we're using this PR to update the latest release, we shouldn't keep any ` changelog.d/` files around, as the typical release flow has them get removed by the towncrier script.

Reply:
We've agreed to not rewrite release history, so this changelog entry can stay & get handled when the towncrier script cuts the next `CHANGELOG.md` update.